internal/mode/static/status/prepare_requests.go

```diff
@@ -406,6 +406,35 @@ func PrepareBackendTLSPolicyRequests(
 	return reqs
 }
 
+// PrepareSnippetFilterRequests prepares status UpdateRequests for the given SnippetsFilters.
+func PrepareSnippetFilterRequests(
+	snippetFilters map[types.NamespacedName]*graph.SnippetsFilter,
+	transitionTime metav1.Time,
+) []frameworkStatus.UpdateRequest {
+	reqs := make([]frameworkStatus.UpdateRequest, 0, len(snippetFilters))
+
+	for nsname, snippetFilter := range snippetFilters {
+		allConds := make([]conditions.Condition, 0, len(snippetFilter.Conditions)+1)
+
+		allConds = append(allConds, staticConds.NewSnippetFilterAccepted())
```

Comment:
Let's add a comment here describing that order matters. I think the other functions in this file have a similar comment that you can base yours off of.